### ISSUE TITLE
Combining tilde overlay with capital H

### ISSUE THREAD

**Issue by TheKnightWho** [psb1558/Junicode-font#126]
Hi - I've come across this in Statutes, in an 1136 Charter of King Stephen as an abbreviation for "Haec".
![image](https://user-images.githubusercontent.com/6895210/158981990-ba16891a-3817-44a8-baed-bc686fec3852.png)

Having looked at the [original charter](https://earlyenglishlaws.ac.uk/laws/manuscripts/ex/?tp=s&nb=5640), an engraving of which is also copied as a plate in Statutes, it looks like this:
![image](https://user-images.githubusercontent.com/6895210/158982594-65d756fc-9afc-4744-83f3-95f17437dc24.png)

It seems that the Record Commission made the editorial decision to use the modern H majuscule, but in lowercase instances that exact same abbreviation is represented with the combining tilde overlay. This is from the beginning of the same charter:
![image](https://user-images.githubusercontent.com/6895210/158983059-e587cdf5-1fbd-4ff2-83e3-6de6373bc2d8.png)

And in the original:
![image](https://user-images.githubusercontent.com/6895210/158983150-4b2482fb-906d-4cf1-8ea7-96de899f268c.png)

However, at the moment it doesn't work with capital H. Would it please be possible for you to include an anchor for it on the upper right stem?

**Comment by psb1558** [comment]
There's a problem that the anchor that positions the tilde overlay is generally placed in the center of the H for positioning of a bar that crosses both vertical stems. As that is the way to place a bar on H (see U+0126 Ħ, the uppercase partner of U+0127 ħ), would it make sense to treat H with tilde overlay the same way?

Alternatively, one could place the tilde on an "enlarged minuscule" H (that is H + cv15[2]—minuscule in shape, but it is searchable as a cap:
<img width="116" alt="image" src="https://user-images.githubusercontent.com/16215767/159025427-9918f067-1e56-4b3f-a367-830fd9f651b7.png">
That anchor is badly positioned right now, but I've fixed it in the source.

**Comment by TheKnightWho** [comment]
That's a good point - I hadn't realised that. One thing that occurs to me is that you could do an alternative conventional H with a different anchor point (maybe as cv15[3]?), as I'd prefer to avoid modifying the capital to something out of sync with both the rest of the text and Statutes, but I appreciate that it would be quite artificial!

**Comment by psb1558** [comment]
I don't see any precedent for a combining mark on the right stem of H: it doesn't make any sense to me, given that you're not seeing anything like it in your sources.

Here's what I would do. On the analogy of U+0126 Ħ / U+0127 ħ, supply H with long tilde overlay to match the h with tilde overlay:
<img width="98" alt="image" src="https://user-images.githubusercontent.com/16215767/159064703-ab4b6579-9a69-46b6-9c53-8bf636ce538a.png">  <img width="71" alt="image" src="https://user-images.githubusercontent.com/16215767/159065793-97e1e78f-e3e7-467b-b761-790e46466497.png">
And then have ccmp resolve the sequence H U+0334 tilde overlay to that character.

Of course, you can reproduce exactly what's in the printed source with H U+02DC spacing tilde, though that was plainly the printer's desperation move: he didn't have the piece of type he needed and faked it up as best he could.

**Comment by TheKnightWho** [comment]
You're right - I suppose I was trying to find some way of resembling what was on the page while making it clear that it is an overlaid tilde, but it is completely artificial. I'm very happy to go with whatever you decide, and it looks good.

**Comment by psb1558** [comment]
Done now.
